### PR TITLE
merge Weather app to development

### PR DESCRIPTION
**Pull request by fedgut**
- Created logic to fetch API data and handling errors
- Created views to display the weather

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle: 
Hi there @fedgut :hand: 
Great job so far :clap:. You did a really nice job on your app. It's very organized :+1:.
But before moving on there is something we need to change.

- [x] As it said in the instructions, the app should change its appearance based on the data that receives.
Screenshot from 2020-04-15 21-05-26

Please kindly submit for code review again after making this change.

Happy coding :computer: 
**Sergio Torres**

**Comment by fedgut** [comment]
Thank you for the code review! I've added an icon that describes the current weather at the location

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "rules": {},
+    "ignorePatterns": [
+        "dist/app.bundle.js",
+        "node_modules/"
+    ]
+}
```
Comment (Oghenebrume50) [inline .eslintrc.json]: Please do not change the linter file see this

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED

Hi @fedgut good job on working on this project and making corrections as the previous reviewer specified :+1:  I have left a note below, please work on that and request for another review :slightly_smiling_face: 

Also, your UI is really clean, Kudos :+1: 

**RAPHAEL NORIODE** :world_map:

File: .eslintrc.json (hunk 1 above)
Comment (fedgut) [reply .eslintrc.json]: Ohh absolutely right! I forgot to add the recommended microverse config, thank you for noticing!

I am also keeping the ignore patterns, as they are just the result of using webpack and npm, not code written by myself.

**Review by maelfosso — APPROVED** [review]
# Congratulations :clap: 

It's OK :+1:  You can merge it now :smile: 
Good luck with the next :handshake: 

Mael FOSSO

**Pull request merged**